releases/mainnet/1.9.2/abi/index.html

```diff
@@ -0,0 +1,12 @@
+<html>
+	<div>
+		<h2>
+			<strong>Mainnet Release 1.9.2 - ABIs</strong>
+		</h2>
+		<p>Download an <a href="/releases/mainnet/1.9.2/abi/mainnet_abi_1.9.2.zip" target="_blank">archive of all Mainnet 1.9.2 ABIs</a>.<p>
+		<p>&nbsp;</p>
+		<a href="/">
+			<sub>Back to the main page</sub>
+		</a>
+	</div>
+</html>
```

Comment:
This is a significant formatting change relative to 1.9.1:

https://github.com/makerdao/mcd-changelog/blob/master/releases/mainnet/1.9.1/index.html

Unless there's a good reason for this, let's minimize the diff. I guess deploying to IPFS (and hence having the IPFS link) is blocked on the pips, ofc.

Reply:
Wait, nvm, I'm looking at the wrong file.